pkgs/applications/misc/openring/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoModule, fetchFromSourcehut, lib }:
+buildGoModule rec {
+  pname = "openring";
+  version = "master";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = pname;
+    rev = "f13edb5dfd882ce608d61cf6b6740650ce9d84a3";
+    sha256 = "sha256-Z65V77JZ9jCzBg7T2+d5Agxxd+MV2R7nYcLedYP5eOE=";
+  };
+
+  vendorSha256 = "sha256-BbBTmkGyLrIWphXC+dBaHaVzHuXRZ+4N/Jt2k3nF7Z4=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A webring for static site generators";
+    homepage = "https://git.sr.ht/~sircmpwn/openring";
+    license = licenses.gpl3;
```

Comment:
gl3Only or gpl3Plus?

Reply:
I've asked upstream developer on IRC to clarify.

Reply:
Relevant mailing list post: https://lists.sr.ht/~sircmpwn/public-inbox/%3C20210403182413.g64oc5alrgkbkmeh%40coruscant%3E

Reply:
The maintainer clarified that it is `gpl3Only`. https://lists.sr.ht/~sircmpwn/public-inbox/%3C20210403182413.g64oc5alrgkbkmeh%40coruscant%3E#%3CCAEDVDIBZ5PX.1YXHCZQG995MC@taiga%3E